meetings/2022-11/nov-30.md

```diff
@@ -0,0 +1,968 @@
+
+USA: First up, we have Ramadan.
+
+Wow. don't need to talk about this at just suggesting different set of flies a
+
+USA: All right. Next we have YSV.
+
+YSV: Yeah, I just want to emphasize that the important change here is that we are now using a namespace. space rather than having this new concept of the lazy. Bindings that will then replace themselves when we do the first access of those bindings. And I think that this or at least I hope and this is what I would like to have feedback from, for example, from Jordan, who raises some the previous meeting, whether or not this would be sufficient. Because this would now be effectively a getter on a namespace. So that's the first piece I have and I can go to my next item.
+
+USA: before that RPR has a reply to that.
+
+RPR: Yeah, I think this, this change the proposal. I understand why it's been made and I'm generally in favor, you know, anything that pushes this proposal for words is amazing, The main thing is it's kind of like a small loss of ergonomics because one of the principles of this small proposal, it's something. where you can make a surgical change to change the form of performance characteristics of loading. Ideally, it's an optimization that you sprinkle in. And the whole reason is that this is much easier and better than going through and making all your code base pacing, which is a very non-ergonomic and Rippling effect. Whereas with this way, this kind of thin means now now that Probably will have you will have more work to do. than just adding a keyword because you'll then have to go through all of the usage sites and prefix of the namespace. So it slightly moves it away from the goal. but it's you know, definitely not make it.  \
```

Comment:
```suggestion
RPR: I think this changes the proposal. I understand why it's been made and am generally in favor. Anything that pushes this proposal forwords is amazing. The main thing is it's a small loss of ergonomics because one of the principles of this proposal is that it's something where you can make a surgical change to the performance characteristics of loading. Ideally it's an optimization that you sprinkle in. And the whole reason is that this is much easier than going through and making all your code's callstack async, which is a very non-ergonomic and rippling workaround. Whereas with this change (the loss of named imports) this means developers will have more work to do compared to just adding a keyword, because you'll then have to go through all of the usage sites and prefix them with a namespace (`ns.identifier`). So it slightly moves the proposal away from that goal. But it's definitely not a showstopper.
```